src/html/login/index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+require('/var/www/app/pages/Layouts.php');
+require('/var/www/app/pages/Components.php');
+
+layoutStart("ログイン|ひとこと掲示板", "ログイン");
+
+?>
+
+<form>
+
+<?php
+
+cInputSection("メールアドレス", "user", "text", []);
+cInputSection("パスワード", "password", "text", []);
```

Comment:
```suggestion
cInputSection('メールアドレス', 'email', 'email');
cInputSection('パスワード', 'password', 'password');
```

- 空配列なら第四引数書かなくてOK
- メールアドレスはtype属性とname属性をemailってするのがいいかも
- パスワードもtype属性はpasswordで
- PHPの文字列は基本シングルクオートで統一したい、他のところも'にしてもらえるとgood

まあ見た目だけだから別にいいかとも思うけど、一応修正お願いー
疑問点あれば聞いてー